cmd/cluster-init/main.go

```diff
@@ -92,10 +94,19 @@ func validateOptions(o options) []error {
 			}
 		}
 
-		if o.clusterName != "" {
-			buildDir := buildFarmDirFor(o.releaseRepo, o.clusterName)
-			if _, err := os.Stat(buildDir); !os.IsNotExist(err) {
-				errs = append(errs, fmt.Errorf("build farm directory: %s already exists", o.clusterName))
+		if o.update {
+			if o.clusterName != "" {
+				buildDir := buildFarmDirFor(o.releaseRepo, o.clusterName)
```

Comment:
https://github.com/openshift/ci-tools/pull/2378/files#diff-5b92b4f3254b078d3a1a836f7055b073b60ddc08f024494639503b0052188d8dR247

We could do more detailed checks like ^.

Reply:
Again, instead of *validating* the content, just generate the correct content and merge.

Reply:
Build farm directory updating will come in the next PR. I will add the relevant ones with that.